charts/snowplow-iglu-server/templates/iglu-hooks.yaml

```
@@ -19,10 +19,11 @@ spec:
         imagePullPolicy: Always
 
         args:
-        - "-Dconfig.override_with_env_vars=true"
         - "setup"
 
         env:
+        - name: "JDK_JAVA_OPTIONS"
+        - value "-Dconfig.override_with_env_vars=true"
```

```suggestion
          value: "-Dconfig.override_with_env_vars=true"
```

The `name` + `value` is a pair so there cannot be a dash in front of it.